sympy/functions/special/gamma_functions.py

```diff
@@ -943,7 +943,7 @@ def _eval_rewrite_as_intractable(self, z, **kwargs):
         return log(gamma(z))
 
     def _eval_is_real(self):
-        return self.args[0].is_real
+        return self.args[0].is_positive
```

Comment:
Are there any cases where `z.is_positive` is False but `loggamma(z).is_real` should be True? If so then it would be better to do:
```
if self.args[0].is_positive:
    return True
```
(and add a test that this does not give False in that case)

Reply:
Yes, that's a good point. For example, `0.306760348727326 + 3*i` (approximately). I don't think these points have any closed form representation so I'm not sure what a good test would be.